### DISCUSSION TITLE
Add ability to select and expose a camera as a generic IP camera?

### DISCUSSION THREAD

**Discussion by rcampbel3** [timmo001/system-bridge#669] [Ideas]
Just discovered system-bridge, and am excited about the potential and possibilities. Would it also be possible to define a PC-attached camera as a dedicated IP camera that HA can use? Maybe using a virtual device so it could be non-exclusive use as well? Thanks for the consideration.

**Comment by timmo001** [comment]
To do this it would require the camera to be in constant use, and would remove access for other applications to the camera. If there is a solution to this, I can't find one
